openc3-cosmos-script-runner-api/scripts/running_script.py

```diff
@@ -1223,7 +1225,7 @@ def output_thread_body(self):
         while True:
             if RunningScript.cancel_output:
                 break
-            if (time.time() - self.output_time) > 5.0:
+            if (time.time() - self.output_time_value) > 5.0:
```

Comment:
Did my refactor break this or was it always broke?

Reply:
I don't know.